test/Driver/actions.swift

```diff
@@ -39,9 +39,8 @@
 // RUN: %swiftc_driver -driver-print-actions -gnone -g %s 2>&1 | %FileCheck %s -check-prefix=DEBUG
 // DEBUG: 0: input, "{{.*}}actions.swift", swift
 // DEBUG: 1: compile, {0}, object
-// DEBUG: 2: merge-module, {1}, swiftmodule
-// DEBUG: 3: link, {1, 2}, image
-// DEBUG: 4: generate-dSYM, {3}, dSYM
+// DEBUG: 2: link, {1}, image
+// DEBUG: 3: generate-dSYM, {2}, dSYM
```

Comment:
Hm. Looking at this I think `-g` + linking + `.swift` inputs should still result in a swiftmodule being generated. Otherwise, you have to specify `-emit-module` as a primary output even if you don't need one.